.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+profile=black
+known_first_party= ansiblelint
```

Comment:
I don't want random undocumented settings here. Also you missed the request to inherit from https://github.com/ansible/pylibssh/blob/c83d151/.isort.cfg:
```suggestion
default_section = FIRSTPARTY
indent = '    '
# Should be: 80 - 1
line_length = 79
# https://github.com/timothycrosley/isort#multi-line-output-modes
multi_line_output = 5
known_first_party = ansiblelint
```

Reply:
No undocumented options present at this point.